src/Microsoft.AspNet.Razor.Runtime/IBufferedStringCollection.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Razor.Runtime
+{
+    /// <summary>
+    /// Contract for a buffered string collection.
+    /// </summary>
+    public interface IBufferedStringCollection
```

Comment:
The interface is just to safeguard the API in-case we choose to change the implementation. This is not used for anything non-TagHelper related.